eval/src/vespa/eval/tensor/dense/dense_xw_product_function.cpp

```diff
@@ -92,7 +92,9 @@ eval::InterpretedFunction::op_function my_select2(CellType lct, CellType rct) {
             return my_cblas_float_xw_product_op<common_inner>;
         }
     }
-    return select_2<MyXWProductOp<common_inner>>(lct, rct);
+    using Target = MyXWProductOp<common_inner>;
+    using MyTypify = eval::TypifyCellType;
+    return typify_invoke<2,MyTypify,Target>(lct, rct);
```

Comment:
could probably typify more? (common_inner)

Reply:
please use same strategy as suggested for matmul in
https://github.com/vespa-engine/vespa/pull/13571